bot/exts/fun/duck_game.py

```diff
@@ -296,8 +307,7 @@ async def end_game(self, channel: discord.TextChannel, game: DuckGame, end_messa
             missed_text = "Flights everyone missed:\n" + "\n".join(f"{ans}" for ans in missed)
         else:
             missed_text = "All the flights were found!"
-
-        await game.embed_msg.edit(content=f"{missed_text}")
+        await self.append_to_found_embed(game, f"\n{missed_text}")
```

Comment:
I would prefer to keep the empty line here if possible

Reply:
The end_game function has two purposes: sending a scoreboard embed and editing the flights-found embed. That line is the culmination of the second process, so it's grouped with the other lines involved in that process.